ember-cli-build.js

```diff
@@ -103,23 +103,9 @@ function routeRecognizer() {
 }
 
 function router() {
-  // TODO upstream this to router.js and publish on npm
-  return new Rollup('bower_components/router.js/lib', {
+  return new Rollup(path.resolve(path.dirname(require.resolve('router_js')), '../lib'), {
```

Comment:
Yes, it can just be `require('router_js/package')['jsnext:main']` at that point.  I thought though they were pushing 'module' now instead of 'jsnext:main'?